app/views/admin/_admin_nonevent_points_manager.html.erb

```diff
@@ -1,5 +1,18 @@
 <%# app/views/points/_admin_nonevent_points_manager.html.erb %>
 
+<h1>Award Points</h1>
+
+<% if flash[:notice] %>
```

Comment:
Should not need this in the partial since this exact code is already in the admin index.html.erb where this part is rendered. Adding this here would show the flash message twice.

Reply:
I made the change